lib/parsers.js

```diff
@@ -83,9 +95,11 @@ exports.parseError = function (err, agent, cb) {
       if (match) error.exception.code = match[1]
     }
 
-    var props = stackman.properties(err)
-    if (props.code) delete props.code // we already have it directly on the exception
-    if (Object.keys(props).length > 0) error.exception.attributes = props
+    if (captureAttributes) {
+      var props = stackman.properties(err)
+      if (props.code) delete props.code // we already have it directly on the exception
+      if (Object.keys(props).length > 0) error.exception.attributes = props
+    }
```

Comment:
Just saying outloud that `error.exception.attributes` won't be at all if `captureAttributes` is false.  Is is probably OK since this would already be the case if `Object.keys(props)` was zero-length.